dm/tracer/config.go

```diff
@@ -94,7 +94,7 @@ func (c *Config) Parse(arguments []string) error {
 func (c *Config) String() string {
 	cfg, err := json.Marshal(c)
 	if err != nil {
-		log.Errorf("[tracer] marshal config to json error %v", err)
+		logger.Error("marshal tracer config to json", log.ShortError(err))
```

Comment:
use `log.L()` or `logger`? if use ``log.L(), update for _dm/tracer/http_handler.go_ and _dm/tracer/http_status.go_ too.